src/components/features/best-selling-section/components/left-side-best-selling/left-side-best-selling.tsx

```
@@ -0,0 +1,27 @@
+            {/* REUSABLE SUBTITLE */}
+            <Subtitle text="Best Selling" />
+            {/* HEADING */}
+            <h2 className={cn("text-maroon-700 dark:text-softpink-200 text-3xl font-bold")}>
+                <span className={cn("text-softpink-500")}>Check Out</span> What <br />
+                Everyone’s <span className={cn("text-softpink-500")}>Buying</span> Right Now
+            </h2>
+            {/* DESCRIPTION */}
+            <p className={cn("text-base text-zinc-500 dark:text-zinc-400")}>
+                Not sure what to choose?
+                <br />
+                Start with our best sellers, these are the gifts our customers keep coming back for. Whether you're celebrating a birthday,
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)